dev-tools/packaging/packages.yml

```
@@ -173,6 +173,11 @@ shared:
     extra_vars:
       repository: 'docker.elastic.co/beats'
 
+  - &docker_ubi_spec
+    extra_vars:
+      image_name: '{{.BeatName}}-ubi'
```

```suggestion
      image_name: '{{.BeatName}}-ubi7'
```
The image should be suffixed `-ubi7` instead of `-ubi` (https://github.com/elastic/dev/issues/1287#issuecomment-548167410)

---

I've updated the name to end with `-ubi7`.